portal-frontend/src/styles/colors.scss

```diff
@@ -1,4 +1,4 @@
-$primary-color: #065a2f;
+$primary-color: #065A2F;
```

Comment:
Please leave lowercased

Reply:
will do, looks like I need to update the settings, sets to uppercase automatically